### ISSUE TITLE
Radarr fails to download from Usenet if you add a Torznab indexer with Jackett switched off.

### ISSUE THREAD

**Issue by danjames92** [Radarr/Radarr#1866]
Steps to reproduce:

- Setup Jackett.
- Add Indexer to Radarr via Torznab.
- Have an already working Usenet setup.
- Set Prefer Usenet to ON (probably not required but this is how I found it).
- Turn Jackett off, leave torznab indexer in Radarr.
- RSS Sync falls to its knees. ;) 
- Watch while grabs fail to happen UNLESS you search via manual download and snatch yourself / freak out cause you worry a movie might die before you realize. ;)

Radarr Version: 0.2.0.778

Log Snippet:

![chrome_2017-08-03_19-24-39](https://user-images.githubusercontent.com/16107603/28937033-708687d0-7881-11e7-8443-ed72b0406043.png)

**Comment by evanrich** [comment]
so because jackett wasn't on to help search, you got an error?  I got the same error when I changed my jackett IP...and forgot to update it.  Not sure how you expect it to search via jackett if jackett is off?

**Comment by danjames92** [comment]
No you don't understand the problem. Re-read to what I wrote.

Jackett being turned off is not the problem.

Jackett being turned off (with a torrent indexer listed on Radarr) causes Radarr to not search via USENET at all. If a torrent indexer doesn't respond, this should not stop a usenet indexer from grabbing results.

On 6 Aug 2017, at 07:46, Evan Richardson <notifications@github.com<mailto:notifications@github.com>> wrote:


so because jackett wasn't on to help search, you got an error? I got the same error when I changed my jackett IP...and forgot to update it. Not sure how you expect it to search via jackett if jackett is off?
